tests/NPSP.robot

```diff
@@ -141,33 +157,32 @@ Create HouseHold
     [return]                  ${account_id}
 
 Create Primary Affiliation
-    # Create Organization Account
-    ${account_id} =  Create Organization Foundation
-    &{account} =  Salesforce Get  Account  ${account_id}
-    
-    # Create Contact
-    ${contact_id} =  Create Contact with Email
-    &{contact} =  Salesforce Get  Contact  ${contact_id}   
+    [Arguments]      ${acc_name}      ${con_id}
+    # # Create Organization Account
+    # &{account} =  API Create Organization Account
+    # # Create Contact 
+    # &{contact} =  API Create Contact    Email=skristem@robot.com
+    Go To Record Home  ${con_id}
     Select Tab    Details
-    Scroll Page To Location    0    300
+    #Scroll Page To Location    0    300
     Click Edit Button    Edit Primary Affiliation
-    Populate Lookup Field    Primary Affiliation    &{account}[Name]
+    Populate Lookup Field    Primary Affiliation    ${acc_name}
     Click Record Button    Save 
-    [Return]         ${account_id}    ${contact_id}   
+    # [Return]         &{account}[Id]    &{contact}[Id]   
 
 Create Secondary Affiliation
-    # Create Organization Account
-    ${account_id} =  Create Organization Foundation
-    &{account} =  Salesforce Get  Account  ${account_id}
-    
-    # Create Contact
-    ${contact_id} =  Create Contact with Email
-    &{contact} =  Salesforce Get  Contact  ${contact_id}   
+    [Arguments]      ${acc_name}      ${con_id}
+    # # Create Organization Account
+    # &{account} =  API Create Organization Account
+    # # Create Contact
+    # &{contact} =  API Create Contact    Email=skristem@robot.com
+    Go To Record Home  ${con_id}
+    Wait For Locator    record.related.title    Volunteer Hours 
     Scroll Page To Location    50    400
     Click Related List Button   Organization Affiliations    New
-    Populate Lookup Field    Organization    &{account}[Name]
+    Populate Lookup Field    Organization    ${acc_name}
     Click Modal Button    Save
-    [Return]         ${account_id}    ${contact_id}
+    #[Return]         &{account}[Id]    &{contact}[Id]
```

Comment:
Are these keywords still used or is everything using the API version now?

Reply:
these keywords are still being used in the test cases that actually tests create contact, create org, create primary affliation etc. In the test cases where these are pre requisites for creating the actual test case, I am using API version.

Reply:
That makes sense, and explains why we no longer need the Return. Thanks.

My preference would be to remove code entirely rather than leaving commented code, since we can always find older versions of the code in github. However I can understand if you'd rather keep bits of commented code around while this work is still in progress, so I'll leave it up to you.

Reply:
I am not that good with git so left those as comments for now since we are still making changes. Once we finalize all the code, I will remove the comments.